Makefile

```diff
@@ -25,7 +30,7 @@ XDR_FILES_NEXT= \
 	Stellar-contract-config-setting.x
 XDR_FILES_LOCAL_NEXT=$(addprefix xdr/next/,$(XDR_FILES_NEXT))
 
-XDRGEN_COMMIT=8d303b1
+XDRGEN_COMMIT=master
```

Comment:
is there a reason not to pin xdrgen runtime to a specific hash/tag?

Reply:
its usually always been master, just got changed when we had the weird `set_` bug so I could do xdrgen w/o needing that PR merged